lib/checks/aria/required-children.js

```diff
@@ -109,7 +112,12 @@ if (!missing) {
 
 this.data(missing);
 
-if (reviewEmpty.includes(role)) {
+// Only review empty nodes when a node is both empty and does not have an aria-owns relationship
+if (
+	reviewEmpty.includes(role) &&
+	node.children.length === 0 &&
+	idrefs(node, 'aria-owns').length === 0
```

Comment:
Can you add a test to show that this returns undefined if there are no children, but false when it has children that don't have the correct rule?

Reply:
👍